src/components/playground/TypographyInput.js

```diff
@@ -13,6 +13,19 @@ import {
 } from '../../context/playgroundContext';
 import { propRules } from '../../models/themePlaygroundOptions';
 
+const allowedFontWeights = new Set([
+  'normal',
+  'bold',
+  'BloodtypeOutlined',
```

Comment:
I do not see this font weight on the Mozilla page https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight